server/app/controllers/login.py

```diff
@@ -52,16 +52,24 @@ def get_id(self):
 @app.route("/login", methods=["GET", "POST"])
 def login():
 
+    repository = Repository()
+
     if current_user.is_authenticated:
-        return render_template("login.html", user = current_user)
+        stations = repository.owned_stations(current_user.get_id())
+
+        # list of stations
+        l = ""
+        for s in stations:
+            l += s['name'] + "(" + str(s['station_id']) + ") "
```

Comment:
Python support just string interpolation and you can write:
```
l += f"{s['name']}({s['station_id']})"
```
You shouldn't concat strings in the loop, it is ineffective. 
You can use ```" ".join(iterable)``` function and list comprehension:
```
l = " ".join(f"{s['name']}({s['station_id']})" for s in stations)
```

Reply:
Awesome! I knew there's a more pythonic way to do it.